src/elements/content-sidebar/activity-feed/common/datestamp/Datestamp.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from 'react';
+import ReadableTime from '../../../../../components/time/ReadableTime';
+import { MILLISECONDS_PER_YEAR } from '../../../../../constants';
+import type { ISODate } from '../../../../../common/types/core';
+
+export type Props = {
+    date: ISODate | Date | number,
+};
+
+const Datestamp = ({ date, ...rest }: Props) => {
```

Comment:
Should we name this something like `ActivityDatestamp` to stay consistent with its Timestamp equivalent?